styles.css

```diff
@@ -419,3 +419,304 @@ form {
   text-align: center;
   margin-left: 125px;
 }
+
+@media (min-width: 768px) {
+  /* .............header section.......... */
```

Comment:
- [ ] As stated by the previous reviewer, kindly adjust your media queries to have the same design as that given on the figma template. according to the requirements,, your desktop version should be responsive from the screen size of `768px`, but this is what yours look like below :
![image](https://user-images.githubusercontent.com/50990925/220799608-a0f8149a-38f1-4569-9e61-8e489c4384bc.png)

## Expected based on figma template:
![image](https://user-images.githubusercontent.com/50990925/220800098-4d02b8ce-fcdc-4b68-92b8-7d81aef06f71.png)



## Requirements :
![image](https://user-images.githubusercontent.com/50990925/220799820-0bfbaf3f-3129-43f5-924d-5a5a6be9511e.png)

- This [resource](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) might be helpful 👍